components/LikerList.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <div class="liker-list">
+    <div class="lc-container-2">
+      <like-form>
+        <template slot="header-right">
+          <a
+            class="lc-underline"
+            href="https://like.co/"
+            rel="noopener noreferrer"
+            target="_blank"
+          >{{ $t('LikerList.label.aboutLikeCoin') }}</a>
+        </template>
+
+        <span
+          v-if="hasFetchedListDetails"
+          class="lc-color-like-dark-brown-2 lc-font-size-20"
+        >
+          {{ $t('Embed.label.numLikesForArticle', {
+            numOfLikees: likers.length,
+            numOfLikes,
+          }) }}
+        </span>
+
+        <div
+          :class="['liker-list__wrapper', 'lc-margin-top-8', { expand: isShowAll }]"
+          :style="{ maxHeight: `${Math.ceil(likers.length / 2) * 74}px` }"
+        >
+
+          <user-avatar
+            v-for="(liker, index) in likers"
+            :key="index"
+            :user="getUserMinInfoById(liker)"
+          />
+
+          <transition name="lc-transition-default">
+            <div
+              v-if="!isShowAll"
+              class="overflow-overlay"
+            />
+          </transition>
+        </div>
+
+        <div
+          class="lc-margin-top-12 lc-text-align-center"
+        >
+          <md-button
+            v-if="!isShowAll"
+            class="lc-color-like-green lc-underline"
+            @click="isShowAll = true"
+          >
+            {{ $t('Embed.button.showMore') }}
+          </md-button>
+        </div>
+      </like-form>
+    </div>
+  </div>
+</template>
+
+
+<script>
+import { mapActions, mapGetters } from 'vuex';
+
+import LikeForm from '~/components/LikeForm';
+import UserAvatar from '~/components/UserAvatar';
+
+export default {
+  name: 'liker-list',
+  components: {
+    LikeForm,
+    UserAvatar,
+  },
+  props: {
+    id: {
+      type: String,
+      required: true,
+    },
+    referrer: {
+      type: String,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      isShowAll: true,
+    };
+  },
+  computed: {
+    ...mapGetters([
+      'getLikerListDetails',
+      'getUserMinInfoById',
+    ]),
+    hasFetchedListDetails() {
+      return this.getLikerListDetails(this.referrer).likers !== undefined;
+    },
+    likers() {
+      return this.getLikerListDetails(this.referrer).likers || [];
+    },
+    numOfLikes() {
+      return this.getLikerListDetails(this.referrer).numOfLikes || 0;
+    },
+  },
+  watch: {
+    likers(list) {
+      this.isShowAll = list.length <= 8;
+    },
+  },
+  mounted() {
+    const { id, referrer } = this;
+    this.fetchLikeButtonStatistics({ id, referrer });
```

Comment:
since `getLikerListDetails` is already in store, is there anyway to prevent repeated fetching?
or is repeated fetching intentional as a update mechanism?